bin/node/src/commands/genesis/inputs.rs

```diff
@@ -25,6 +25,8 @@ pub struct BasicWalletInputs {
     pub init_seed: String,
     pub auth_scheme: AuthSchemeInput,
     pub auth_seed: String,
+    #[serde(default = "_default_true")]
+    pub on_chain: bool,
```

Comment:
Instead of a boolean flag, should we use `AccountStorageType` here?

Also, what is the motivation setting the default value to true?

Reply:
> Instead of a boolean flag, should we use AccountStorageType here?

That can work as well, just seemed simpler to use a bool

> Also, what is the motivation setting the default value to true?

Oh, now that you mention it I wanted to set it as false by default (that is so current configs wouldn't break and would behave as always)

Reply:
I'd prefer to set all the required properties explicitly (and I think it's OK to break things for now).

Reply:
should be changed now